src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -775,6 +778,97 @@ const CommandParametersPermissions& CommandRequestImpl::parameters_permissions()
   return parameters_permissions_;
 }
 
+void CommandRequestImpl::StartAwaitForInterface(
+    const HmiInterfaces::InterfaceID& interface_id) {
+  awaiting_response_interfaces_.insert(interface_id);
+}
+
+bool CommandRequestImpl::GetInterfaceAwaitState(
```

Comment:
@LuxoftAKutsan 
From method name I would expect to get specific state returned but instead it returns bool. I think better to rename method to IsInterfaceAwaited() or to add typedefs for true/false like 
AWAITING/NOT_AWAITING